App Package - Enabling Basic Requirements Definition (#85)
margo/specification: system-design/margo-api-reference/workload-api/application-package-api/application-description.md

| Attribute | Type | Required? | Description |
| --- | --- | --- | --- |
| type | string |  Y  | Defines the type of this deployment configuration for the application.  The allowed values are `helm.v3`, to indicate the deployment profile's format is Helm version 3,  and `compose` to indicate the deployment profile's format is a Compose file.  When installing the application on a device supporting the Kubernetes platform, all `helm.v3` components,  and only `helm.v3` components, will be provided to the device in same order they are listed in the application description file.  When installing the application on a device supporting Compose, all `compose` components,  and only `compose` components, will be provided to the device in the same order they are listed in the application description file.  The device will install the components in the same order they are listed in the application description file.|

> | id | string |  Y  | An identifier for the deployment profile, given by the application developer, used to uniquely identify this deployment profile in scope of this application description.  The id can be used to display deployment profiles in a user interface and being able to unique distinguish each one of them.|

Reviewer: ```suggestion
| id | string |  Y  | An identifier for the deployment profile, given by the application developer, used to uniquely identify this deployment profile from others within this application description's scope. |
```

Author: Thanks, I changed it accordingly.
